changelogs/fragments/441-add-galaxy-requirement-file.yml

```
@@ -0,0 +1,3 @@
+---
+major_changes:
```

The dependency is already there, so this is not really a major change :)
```suggestion
minor_changes:
```